rest-backend/src/main/java/org/eclipse/steady/backend/rest/CoverageController.java

```diff
@@ -77,12 +74,12 @@
    *
    * @return a {@link org.springframework.web.servlet.DispatcherServlet} object.
    */
-  @Bean(name = DispatcherServletAutoConfiguration.DEFAULT_DISPATCHER_SERVLET_BEAN_NAME)
-  public DispatcherServlet dispatcherServlet() {
-    DispatcherServlet dispatcherServlet = new DispatcherServlet();
-    dispatcherServlet.setDispatchOptionsRequest(true);
-    return dispatcherServlet;
-  }
+  //	@Bean(name = DispatcherServletAutoConfiguration.DEFAULT_DISPATCHER_SERVLET_BEAN_NAME)
```

Comment:
I would either delete it altogether or add a comment when and why it was commented out